modules/price/src/TwigExtension/PriceTwigExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\commerce_price\TwigExtension;
+
+use Drupal\commerce_price\Price;
+use Drupal\Core\Template\TwigExtension;
+
+/**
+ * Provides a twig filter to be applied to a price object or array.
```

Comment:
Leave room for future filters by only documenting something like "Provides Price-specific Twig extensions."
The code example can be moved to the formatPrice docblock.